### PR TITLE
Add aria2gui v1.3.5

### PR DESCRIPTION
**Pull request by melotusme**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/aria2gui.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (miccal) [inline Casks/aria2gui.rb]: Please remove this empty line.

File: Casks/aria2gui.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+cask 'aria2gui' do
+  version '1.3.5'
+  sha256 'b5a7f0dd34b779cc6cbda6f612ecda6bc57022f30fd371c8433d47eab26cb944'
+
+  url "https://github.com/yangshun1029/aria2gui/releases/download/#{version}/Aria2GUI-v#{version}.zip"
```
Comment (miccal) [inline Casks/aria2gui.rb]: Please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/aria2gui.rb (hunk 2 above)
Comment (melotusme) [inline Casks/aria2gui.rb]: https://github.com/yangshun1029/aria2gui

Comment (melotusme) [inline Casks/aria2gui.rb]: this is the project url

Comment (miccal) [inline Casks/aria2gui.rb]: Please read the documentation I linked to, it will explain how to add an `appcast` and `checkpoint` for a Github release.

Comment (melotusme) [inline Casks/aria2gui.rb]: will this `
appcast 'https://github.com/yangshun1029/aria2gui/releases.aria2gui',
 checkpoint:'36a9e7f1c95b82ffb99743e0c5c4ce95d83c9a430aac59f84ef3cbfab6145068'
` be ok?

Comment (miccal) [inline Casks/aria2gui.rb]: @vitorgalvao has already added it for you.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**